tcms/core/tests/test_ajax.py

```diff
@@ -193,14 +119,14 @@ def setUpTestData(cls):
     def test_render_plan(self):
         response = self.client.get(self.url, {
             'plan': self.test_plan.plan_id
-        })
+        }, follow=True)
```

Comment:
Why do you need `follow=True` here? It is usually used when there is a redirect.